sdk/identity/identity/review/identity.api.md

```diff
@@ -115,6 +115,18 @@ export interface InteractiveBrowserCredentialOptions extends TokenCredentialOpti
     tenantId?: string;
 }
 
+// @public
```

Comment:
I assume changes outside `pnpm-lock.yaml` are accidental and should be reverted?

Reply:
so the build changed the api review file for identity for me and wanted someone from the team to review this.

Reply:
These must have been missed from an Identity PR that went in over the last couple days. Maybe we can leave it out of this PR and @jonathandturner can include it with his next PR?

Reply:
Please move this change to a separate PR.

Reply:
Missing API review changes should break PR validation. Tracking issue: https://github.com/Azure/azure-sdk-for-js/issues/7451

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8037 - created a separate PR